### ISSUE TITLE
Mutation to delete a beacon on leader's demand

### ISSUE THREAD

**Issue by ItsAdityaKSingh** [CCExtractor/beacon-backend#97]
There is no mutation to delete a beacon on leader's demands. Any beacon created gets stored forever. A mutation for deletion could be introduced.

**Comment by aadibajpai** [comment]
> There is no mutation to delete a beacon on leader's demands. Any beacon created gets stored forever. A mutation for deletion could be introduced.

not sure of use case and needs to be thought out more eg does the beacon disappear for all users or what

**Comment by ItsAdityaKSingh** [comment]
Yes, for now what I thought of was say, if user has created some hike by mistake or wants to clear out his list of beacons showing on his home screen. So the user should be given the option to do so.
